plugins/adaptivegrain/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, fetchgit, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "adaptivegrain";
+  version = "0.3.1";
+
+  src = fetchgit {
+    url = "https://git.kageru.moe/kageru/${pname}";
+    rev = version;
+    sha256 = "1sds08r7as23x7kgfa0w7sqkmzpc4xbccirxjwynybcm25ixpzb4";
+  };
+
+  cargoSha256 = "1fy2n6afn7hxkwxf8l6jdakqpq97rn5dqav1bagzm2n40m37ldfk";
```

Comment:
I get a hash mismatch:
```
         specified: sha256-0zV6RgXEivqfWmEr3IrNJ+GLp2rSUOQ6nx0e65Sxwrs=
            got:    sha256-MBroo4JABKP57JxWnDclZF9BuRRJr1R8uZVTljfF1uk=
```

Reply:
What nixpkgs version are you using? `85f96822a05180cbfd5195e7034615b427f78f01` changed how the vendoring is done, so all `cargoSha256` hashes have to be replaced.